src/main/java/com/github/dadrus/App.java

```diff
@@ -6,12 +6,17 @@
  */
 public class App 
 {
+	public static final String PASSWORD="boo";
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
     }
 	
 	public void test() throws Exception {
-		
+		if(true) { // sonar doesn't like this comment style
+			throw new Exception("this"); // and the raw exception anyway
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)